artifacts/definitions/Server/Import/CuratedSigma.yaml

```diff
@@ -12,8 +12,8 @@ required_permissions:
 - SERVER_ADMIN
 
 parameters:
-  - name: PackageName
-    type: choices
+  - name: PackageNames
+    type: multichoice
     default: Velociraptor Hayabusa Ruleset
```

Comment:
@scudette Do you think there should be a default value? The multichoice selector doesn't show the default value so it might be better to have nothing as default.

Reply:
It should show the default value - it needs to be encoded as json array though because that is what the selector uses

Reply:
Ok thanks. I've updated the paramter and tested using only the default value and it seems fine.